platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/ConfigurationCacheAction.kt

```diff
@@ -39,7 +39,10 @@ internal sealed class ConfigurationCacheAction {
      * Configuration cache entry is invalid for the current invocation.
      * The new entry will be stored by the end of the build.
      */
-    object Store : ConfigurationCacheAction() {
-        override fun toString(): String = "Store"
-    }
+    data object Store : ConfigurationCacheAction()
+
+    /**
+     * No valid entry was found and no entry should be created, as CC is in read-only mode.
+     */
+    data object Skip : ConfigurationCacheAction()
```

Comment:
🤔 I like the conciseness of the name, it's in the harmony with other implementations. However, I think it can be confusing to some extend. What exactly do we `skip`? 
I'd say it's `SkipStore`, but I'm not insist

Reply:
Went back to `SkipStore`, as discarding happens with the other actions as well.